lib/mayaUsd/render/mayaToHydra/utils.cpp

```diff
@@ -24,6 +36,10 @@
 
 #include <maya/MGlobal.h>
 
+#if USD_VERSION_NUM >= 2102
+#include <pxr/imaging/garch/glApi.h>
+#endif
```

Comment:
Unlike GLEW, this `garch` include does **not** need to come first.

Reply:
I had a follow-up chat with the Hydra team about this. They were saying that the newly added GL loader doesn't break the way `glew.h` does if it's not `#include`'d first, it's probably still good practice to include it early. I've got a new PR coming shortly with a bunch more instances of this, so I'll revisit this one as part of those changes. I'll also amend the clang-format config so that it includes both `glew.h` and `glApi.h` in the first group.

The Hydra team also just made a change to `GlfContextCaps::InitInstance()` so that it calls `GarchGLApiLoad()` itself, so we'd then be able to remove the call I had to add here for core USD 21.02+.